pcsx2-qt/Settings/SettingsDialog.cpp

```diff
@@ -115,7 +115,7 @@ void SettingsDialog::setupUi(const GameList::Entry* game)
 	{
 		addWidget(m_game_fix_settings_widget = new GameFixSettingsWidget(this, m_ui.settingsContainer), tr("Game Fix"),
 			QStringLiteral("close-line"),
-			tr("<strong>Game Fix Settings</strong><hr>Gamefixes can work around incorrect emulation in some titles.<br>However, they can "
+			tr("<strong>Game Fix Settings</strong><hr>Game fixes can work around incorrect emulation in some titles.<br>However, they can "
```

Comment:
Game Fixes is the whole name we gave it, so technically Fixes should also be capitalised.

Reply:
Now that I see this... Shouldn't the "Game Fix" section (within a game's settings) be renamed to "Game Fixes"?

Reply:
I would say so, yes.